package.json

```diff
@@ -62,24 +65,29 @@
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "1.7.0",
+    "node-sass": "^3.10.1",
     "phantomjs-polyfill": "0.0.2",
     "phantomjs-prebuilt": "^2.1.7",
     "raw-loader": "0.5.1",
     "reflect-metadata": "0.1.2",
     "remap-istanbul": "^0.6.3",
     "rimraf": "^2.5.2",
+    "sass-loader": "^4.0.2",
+    "source-map-loader": "^0.1.5",
+    "style-loader": "^0.13.1",
+    "svg-url-loader": "^1.1.0",
     "ts-helpers": "^1.1.1",
     "tsconfig-lint": "^0.6.0",
     "tslint": "^3.5.0",
     "tslint-loader": "^2.1.0",
     "typescript": "2.0.3",
     "typings": "^1.3.2",
+    "url-loader": "^0.5.7",
     "webpack": "2.1.0-beta.22",
-    "webpack-dev-server": "^2.1.0-beta.2",
-    "source-map-loader": "^0.1.5"
+    "webpack-dev-server": "^2.1.0-beta.2"
   },
   "engines": {
-    "node": ">= 4.2.1 <= 5",
+    "node": ">= 6",
```

Comment:
this seems to be to me the only real issue with the PR. I dont want to force node 6. Do you know where this starts to become a problem? Why is that needed?